frontends/election-manager/src/contexts/app_context.ts

```diff
@@ -33,19 +33,16 @@ export interface AppContextInterface {
   printBallotRef?: RefObject<HTMLElement>;
   saveCastVoteRecordFiles: SaveCastVoteRecordFiles;
   saveElection: SaveElection;
-  setCastVoteRecordFiles: React.Dispatch<
```

Comment:
What if the store was accessible through a separate context? It might be clearer to know which data/functions interact with the store vs which don't. Currently it gets obscured by adding functions to this app context. Might be a bigger change than you want to make for this PR, but wanted to at least get your thoughts.

Reply:
I was thinking of just putting the `store` into the context, or even going through the work of making it so other components could `useElectionManagerStore` to get access to it, but that felt like too much to do in this PR. I agree we should move in that direction, though.